sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/AppendBlobsImpl.java

```diff
@@ -25,19 +25,18 @@
 import com.azure.storage.blob.models.AppendBlobsCreateResponse;
 import com.azure.storage.blob.models.AppendPositionAccessConditions;
 import com.azure.storage.blob.models.BlobHTTPHeaders;
-import com.azure.storage.blob.models.CustomerProvidedKeyInfo;
+import com.azure.storage.blob.models.CpkInfo;
```

Comment:
Don't block the PR on this, but we should make sure we change this name back at some point.

Reply:
This would be a change in the Swagger itself.